istio-operator/istio-operator.spec

```diff
@@ -109,12 +109,19 @@ popd
 install -d $RPM_BUILD_ROOT/manifests
 cp -ra OPERATOR/src/github.com/maistra/istio-operator/manifests-maistra/* $RPM_BUILD_ROOT/manifests
 
+# install the templates
+mkdir -p $RPM_BUILD_ROOT%{_datadir}/istio-operator/default-templates
+cp -ra OPERATOR/src/github.com/maistra/istio-operator/tmp/_output/templates/* $RPM_BUILD_ROOT%{_datadir}/istio-operator/default-templates
+
 %files
 %{_bindir}/istio-operator
 %{_sysconfdir}/istio-operator
+%{_datadir}/istio-operator
 /manifests
 
 %changelog
+* Thu Aug 08 2019 Daniel Grimm <dgrimm@redhat.com> - 1.0.0-2
+- Added templates directory
```

Comment:
Not commenting on the PR itself, let @brian-avery do that. But if you are using -2 here, you will probably want to bump the release number on line 42.

Reply:
Done!

Reply:
Thanks :) my first spec file